arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -390,45 +442,41 @@
             "$ref": "#/parameters/apiVersionParameter"
           },
           {
-            "name": "resourceGroupName",
-            "description": "Name of the resource group within the Azure subscription.",
-            "in": "path",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/resourceGroupNameParameter"
           },
           {
-            "name": "mediaServiceName",
-            "description": "Name of the Media Service.",
-            "in": "path",
-            "required": true,
-            "type": "string",
-            "pattern": "^[a-z0-9]",
-            "minLength": 3,
-            "maxLength": 24
+            "$ref": "#/parameters/mediaResourceNameParameter"
           }
-        ]
-      }
-    },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{mediaServiceName}/syncStorageKeys": {
-      "post": {
-        "summary": "Synchronize the keys for a storage account to the Media Service.",
-        "operationId": "MediaService_SyncStorageKeys",
+        ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "Success. The keys for the Media Services account were listed.",
             "schema": {
-              "$ref": "#/definitions/MediaService"
+              "$ref": "#/definitions/ServiceKeys"
+            },
+            "examples":{
+              "primaryAuthEndpoint": "{primary-auth-endpoint}",

```

Comment:
same as above. Make sure to cover all the review comments made in the first example